interfaces/cython/cantera/reaction.pyx

```diff
@@ -660,6 +660,32 @@ cdef class PlogRate(ReactionRate):
             self._rate.reset(new CxxPlogRate(ratemap))
             self.rate = self._rate.get()
 
+cdef class LinearBurkeRate(ReactionRate):
+    r"""
+    A pressure-dependent reaction rate parameterized by logarithmically
+    interpolating between Arrhenius rate expressions at various pressures.
+    """
+    _reaction_rate_type = "linear-burke"
+
+    def __cinit__(self, rates=None, input_data=None, init=True):
+
+        if init and isinstance(rates, list):
+            self.rates = rates
+
+        elif init:
+            if isinstance(input_data, dict):
+                self._rate.reset(new CxxLinearBurkeRate(py_to_anymap(input_data)))
+            elif rates is None:
+                self._rate.reset(new CxxLinearBurkeRate(py_to_anymap({})))
+            elif input_data:
+                raise TypeError("Invalid parameter 'input_data'")
+            else:
+                raise TypeError("Invalid parameter 'rates'")
+            self.set_cxx_object()
```

Comment:
You should be able to get the constructor from `input_data` working by following the example of the other rate types. This is just a Python data structure containing the same information as the YAML input.

I think you can just remove the `rates` kwarg -- I'm guessing that's just leftover from copying from the PlogRate implementation.

Reply:
I attempted to follow these instructions, and now have a very bare-bones version of it. However, I could use more advice re: what needs to be added now.